packages/syft/src/syft/serde/lib_service_registry.py

```diff
@@ -79,10 +84,13 @@ def __init__(
     def set_signature(self) -> None:
         pass
 
-    def build(self) -> None:
+    def build(self, stack: List, root: Optional[CMPBase] = None) -> None:
```

Comment:
isnt the root always self? the only place where we call .build() is with c.build(stack=self.stack, root=c)

Reply:
yeah, we can change that, I missed it when I moved to bfs